### ISSUE TITLE
wall charger charging cause "go into usb mode" msg

### ISSUE THREAD

**Issue by dracodoc** [koreader/kindlepdfviewer#225]
In file browser or reading mode, plug in wall charger will activate "go into usb mode" msg, and the screen have some messed up displays. kpv still accept input, thus can open file or turn page, then the display will mess up again when the charger is disconnected.

So kpv need to separate pc usb connection and wall charger charging events, either ignore charger event or display some info then refresh the screen.

**Comment by dpavlin** [comment]
@NuPogodi it seems that you fixed this with 925678b1ce0f7b3050cf83771f0bdf705c49c46b but commit message is miss-leading.

I will merge it to master as soon as I get back home to test it with real Kindle (as opposed to emulator)

**Comment by houqp** [comment]
no, it seems that aside from adding screenshot shorcut, NuPogodi manually applied some of my previous changes, not with git pull, so git also treats them as new code.

**Comment by dracodoc** [comment]
The current build still have a little problem with wall charger. There will be a message "go into usb mode" when kindle is plugged in charger, then the screen changed a little bit. Then it actually can respond to operations and can work normally, but it need a manual refresh to clear the message "go into usb mode".

So it seemed that when kpv decided it is in charging instead of connecting to pc, it need to refresh screen to remove that message, or maybe show a message "in charging..." then remove this message too.

**Comment by houqp** [comment]
I think DXG has a charging event that I can capture. Will look into that tomorrow, don't feel like coding tonight :(
